docs/src/pages/components/data-grid/demo/demo.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Data Grid - Demo
+components: DataGrid, XGrid
+---
+
+# Data Grid - Demo
+
+<p class="description">Use the demo below to explore the available features</p>
+
+{{"demo": "pages/components/data-grid/demo/FullFeauredDemo.js", "bg": "inline"}}
```

Comment:
We don't need to show the inline preview, it seems to be noise, we can force it to be collapsed.

Reply:
I don't have a preference. For not it may not be a problem but once there are more demos it will be I think.

Reply:
OK, let's hide the inline preview then, it seems to be a distraction for developers.

Reply:
done